app/components/crate-sidebar.js

```diff
@@ -14,6 +14,10 @@ export default class CrateSidebar extends Component {
     return homepage && (!repository || simplifyUrl(repository) !== simplifyUrl(homepage));
   }
 
+  get cargoAddCommand() {
+    return `cargo add ${this.args.crate.name}`;
```

Comment:
```suggestion
    return `cargo add ${this.args.crate.name}@${this.args.version.num}`;
```

any thoughts about including the version number, like we do for the `Cargo.toml` snippet?

I understand that `cargo add` will automatically use the latest version, but if you're viewing the page of a specific version then maybe it would make sense to include the version in the command?

Reply:
Good point! I didn't consider version pages. I added a commit that adds the version only on version detail pages. The regular crate detail page still shows the versionless variant. That makes the most sense, I think?

Reply:
> Good point! I didn't consider version pages. I added a commit that adds the version only on version detail pages. The regular crate detail page still shows the versionless variant. That makes the most sense, I think?

This is exactly the solution I was imagining when I was considering this idea as `cargo-add` was being merged into cargo.

Thanks for taking this on! I originally held off on getting this done as I wanted more time for versions of `cargo` with `add` to proliferate but lost track of coming back to this.